src/main/java/org/sonarlint/intellij/util/SonarLintUtils.java

```diff
@@ -297,6 +301,30 @@ public static boolean isPhpFile(@NotNull PsiFile file) {
     return "php".equalsIgnoreCase(file.getFileType().getName());
   }
 
+  public static File getIdeIconForOpenInIde() {
+    try {
+      return ((ApplicationInfoImpl) getAppInfo()).getApplicationSvgIconFile();
+    } catch (NullPointerException e) {
+      return null;
```

Comment:
Do we really need to catch this NPE ? Can it happen ? If yes, couldn't we check if something is null instead of dereferencing the null reference ?